### PR TITLE
ROE-992 Display expected value for field -  who is completing the reg…

### PR DESCRIPTION
**Pull request by mattch1**
…istration

### JIRA link

https://companieshouse.atlassian.net/browse/ROE-992

### Change description

Unsure what the check your answers  -> Identity checks -> “Who is completing this registration”

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: test/controllers/check.your.answers.controller.spec.ts [hunk 1]
```diff
@@ -249,6 +249,28 @@ describe("GET tests", () => {
...
+
+  test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with identity check by uk agent`, async () => {
+    const applicationDataWithSomeoneElse = { ...APPLICATION_DATA_MOCK };
+    applicationDataWithSomeoneElse[WhoIsRegisteringKey] = WhoIsRegisteringType.AGENT;
+    mockGetApplicationData.mockReturnValueOnce(applicationDataWithSomeoneElse);
+
+    const resp = await request(app).get(CHECK_YOUR_ANSWERS_URL);
+
+    expect(resp.status).toEqual(200);
+    expect(resp.text).toContain("The UK agent");
```
Comment (mouhajer-ch) [inline test/controllers/check.your.answers.controller.spec.ts:261]: Could you please add text mock [here](https://github.com/companieshouse/overseas-entities-web/blob/main/test/__mocks__/text.mock.ts)

**Review by markpit — DISMISSED** [review]

File: test/controllers/check.your.answers.controller.spec.ts [hunk 2]
```diff
@@ -249,6 +249,28 @@ describe("GET tests", () => {
...
+
+  test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with identity check by someone else`, async () => {
+    const applicationDataWithSomeoneElse = { ...APPLICATION_DATA_MOCK };
+    applicationDataWithSomeoneElse[WhoIsRegisteringKey] = WhoIsRegisteringType.SOMEONE_ELSE;
+    mockGetApplicationData.mockReturnValueOnce(applicationDataWithSomeoneElse);
+
+    const resp = await request(app).get(CHECK_YOUR_ANSWERS_URL);
+
+    expect(resp.status).toEqual(200);
+    expect(resp.text).toContain("Someone from the overseas entity");
```
Comment (mouhajer-ch) [inline test/controllers/check.your.answers.controller.spec.ts:272]: Same [here](https://github.com/companieshouse/overseas-entities-web/blob/main/test/__mocks__/text.mock.ts)

File: test/controllers/check.your.answers.controller.spec.ts (hunk 1 above)
Comment (mattch1) [reply test/controllers/check.your.answers.controller.spec.ts:261]: Not sure what you mean.

Comment (mattch1) [reply test/controllers/check.your.answers.controller.spec.ts:261]: Clicked the link and can see what you mean. As these are only tested in one place I'm doubtful as to whether this is necessary.

Comment (mouhajer-ch) [reply test/controllers/check.your.answers.controller.spec.ts:261]: It is not just reusability but as well maintainability

Comment (mattch1) [reply test/controllers/check.your.answers.controller.spec.ts:261]: Not sure it really adds a lot in that way either what do other people @markpit think?

Comment (markpit) [reply test/controllers/check.your.answers.controller.spec.ts:261]: I'd add it with the other test variables like Mo said to keep the code tidy.

**Pull request merged**